backend/node/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Register the node into admin. This way, we won't need the DELETE API and the server admin can just manage node connections through the /admin endpoint

```suggestion
from .models import Node
admin.site.register(Node)
```